### DISCUSSION TITLE
Add option to start up minimized

### DISCUSSION THREAD

**Discussion by Sans3108** [maximmax42/Discord-CustomRP#164] [Ideas]
As it is right now, the app starts with Windows automatically but it's a little bit annoying that it starts up and shows the GUI, and I have to minimize it manually.

My feature request is this simple, just add an option to make the app start up minimized, without me having to do anything after toggling it on.

**Comment by maximmax42** [comment]
There... is?
In the app there's a top menu row, press `Settings` -> `Start Minimized`.
